src/Frontend/Fragment/Comment.hs

```diff
@@ -22,10 +22,9 @@ import qualified Frontend.Path as U
 
 
 data CommentWidget = CommentWidget
-    { _cwRenderContext :: RenderContext
-    , _cwIdeaCaps      :: [Capability]
-    , _cwComment       :: Comment
-    , _cwPhase         :: Phase
+    { _cwCapCtx   :: CapCtx
+    , _cwIdeaCaps :: [Capability]
```

Comment:
if you document the monotonic property at least as a comment with `FIXME: write tests for this` in LifeCycle, I agree with this change.